locale/zh_CN.ftl

```diff
@@ -67,19 +68,19 @@ future-due-types-help =
 future-due-retention = 预测记忆保留率
 future-due-retention-help = 
     若勾选“按比例显示”，将显示FSRS预测的记忆保留率 (如果你在使用FSRS Helper插件，在卡片浏览器中查看 "target{"\u00A0"}R")；若未勾选，则显示FSRS预测的当日复习卡片数。
-    不计算逾期未复习的卡片
+    不计算逾期未复习的卡片。
 
 pass = 通过
 fail = 失败
 
 intra-day-due = 今日到期分布
 intra-day-due-help =                 
-    此图表显示今日到期卡片的具体小时分布
-    适用于使用自动学习间隔的FSRS-5用户或设置了较长日内学习间隔的情况
+    此图表显示今日到期卡片的具体小时分布。
+    适用于使用自动学习间隔的FSRS-5用户或设置了较长日内学习间隔的情况。
 next-card-at = 下一张卡片将在{ $hours ->
         [0] {""}
-        [one] { $hours }小时
-        *[many] { $hours }小时
+        [one] {" "}{ $hours }小时
```

Comment:
I guess you are trying to add a space before the number? Maybe you need add a space after the number, too.